### PR TITLE
Export fix

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Avoid big accounts to timeout (at 30 seconds) when using the export. 

## Changes

- Add index for account statements on account_id and time.
- Change query to stream data, instead of building a big string.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: backend/Application/Api/Rest/ExportController.cs [hunk 1]
```diff
@@ -42,21 +45,24 @@ public async Task<ActionResult> GetStatementExport(string accountAddress, DateTi
 
         var query = dbContext.AccountStatementEntries
             .AsNoTracking()
-            .Where(x => x.AccountId == account.Id);
+            .OrderByDescending(x => x.Timestamp)
+            .Where(x => x.AccountId == account.Id)
+            ;
```
Comment (orhoj) [inline backend/Application/Api/Rest/ExportController.cs:50]: Move to previous line.

File: backend/Application/Api/Rest/ExportController.cs [hunk 2]
```diff
@@ -42,21 +45,24 @@ public async Task<ActionResult> GetStatementExport(string accountAddress, DateTi
 
         var query = dbContext.AccountStatementEntries
             .AsNoTracking()
-            .Where(x => x.AccountId == account.Id);
+            .OrderByDescending(x => x.Timestamp)
```
Comment (orhoj) [inline backend/Application/Api/Rest/ExportController.cs:48]: Why do you have to order this now when it wasn't needed previously?

File: backend/Application/Api/Rest/ExportController.cs [hunk 3]
```diff
@@ -66,37 +72,34 @@ public async Task<ActionResult> GetStatementExport(string accountAddress, DateTi
...
-            x.Amount,
-            x.AccountBalance
-        });
-        var values = await result.ToListAsync();
-        if (values.Count == 0)
-        {
-            return new NoContentResult();
-        }
+        var result = query.Select(x => string.Format("{0}, {1}, {2}, {3}\n", 
+        x.Timestamp.ToString("u"),
```
Comment (orhoj) [inline backend/Application/Api/Rest/ExportController.cs:76]: Fix formatting.

**Review by orhoj — CHANGES_REQUESTED** [review]

File: backend/Application/Api/Rest/ExportController.cs (hunk 2 above)
Comment (shjortConcordium) [reply backend/Application/Api/Rest/ExportController.cs:48]: Because otherwise it orders by index, and then it doesn't use the index I added.

**Review by orhoj — APPROVED** [review]

**Review by DOBEN — APPROVED** [review]

**Pull request merged**